src/components/Calendar/Body/Body.tsx

```diff
@@ -1,25 +1,47 @@
-import { eachDay, format } from 'date-fns'
 import * as React from 'react'
```

Comment:
I don't know of any reason to do import * as React anywhere.  import React from will work fine and is consistent with other import statements